### ISSUE TITLE
env() conflict when using illuminate/support

### ISSUE THREAD

**Issue by fiskhandlarn** [wordplate/wordplate#254]
I thought I'd make a separate issue for this problem (found in https://github.com/wordplate/wordplate/issues/250).
Whenever a package that requires `illuminate/support` is required in a WordPlate project this leads to `illuminate/support`'s `env()` function being defined before WordPlate's. This is in itself probably good for the user to know. But on top of that, the site will die with a PHP fatal error until `vlucas/phpdotenv` is required.

AFAIK, the WordPlate documentation mentions/recommends these packages creating this issue:
- https://github.com/fiskhandlarn/blade in [Integrations](https://github.com/wordplate/wordplate#integrations) 
- https://github.com/laravel/helpers in [Upgrading from 7 to 8](https://github.com/wordplate/wordplate#upgrade-guide)

## Steps to Reproduce the Problem

```bash
$ composer create-project --prefer-dist wordplate/wordplate blog
$ cd blog/
$ composer require laravel/helpers
$ php -S localhost:8000 -t public/
```

... generates this error in the browser:

```
Fatal error: Uncaught Error: Class 'PhpOption\Option' not found in blog/vendor/illuminate/support/Env.php:81 
```

### Temporary fix
 
```bash
$ composer require vlucas/phpdotenv
```

will remove the error, but will force the project to use `illuminate/support`'s `env()`.

## Versions

  - WordPlate: 8.1.0
  - PHP: PHP 7.3.9 

## Possible solutions

- Renaming/scoping WordPlate's `env()` (but that would be a breaking change, as [vinkla pointed out](https://github.com/wordplate/wordplate/issues/250#issuecomment-626673560))
- Add to the documentation that `vlucas/phpdotenv` needs to required as a prerequisite for the above mentioned packages

**Comment by vinkla** [comment]
Thanks for opening a new issue on the subject @fiskhandlarn 🙌 

This is probably due to a recent switch to `symfony/dotenv` from `vlucas/phpdotenv`. Do you think this issue would be fixed if we switched back?

**Comment by vinkla** [comment]
This has been fixed in version 9.0
